dawn/src/dawn/CodeGen/Cuda/CacheProperties.cpp

```diff
@@ -27,26 +27,44 @@ makeCacheProperties(const std::unique_ptr<iir::MultiStage>& ms,
                     const std::shared_ptr<iir::StencilInstantiation>& stencilInstantiation,
                     const int maxRedundantLines) {
 
-  iir::Extents maxExtents{0, 0, 0, 0, 0, 0};
+  iir::Extents maxExtents(dawn::ast::cartesian_{}, 0, 0, 0, 0, 0, 0);
```

Comment:
here and everywhere else, you can use `dawn::ast::cartesian` instead of `dawn::ast::cartesian_{}`